rtrlib/aspa/ordered_dyn_array/ordered_dyn_array.c

```diff
@@ -7,32 +7,33 @@
 * Website: http://rtrlib.realmv6.org/
 */
 
-#include "alloc_utils_private.h"
 #include "ordered_dyn_array.h"
+#include "rtrlib/aspa/aspa_private.h"
+#include "rtrlib/lib/alloc_utils_private.h"
+
+#define ordered_dyn_array_createIAL_SIZE 128
 
 int ordered_dyn_array_create(struct ordered_dyn_array **vector_pointer)
 {
-	const size_t DEFAULT_INITIAL_SIZE = 128;
-
 	// allocation the chunk of memory of the provider as numbers
-	struct aspas_record *data_field =
-		(struct aspas_record *)lrtr_malloc(sizeof(struct aspas_record) * DEFAULT_INITIAL_SIZE);
+	struct aspa_record *data_field =
+		(struct aspa_record *)lrtr_malloc(sizeof(struct aspa_record) * ordered_dyn_array_createIAL_SIZE);
 
 	// malloc failed so returning an error
-	if (data_field == NULL) {
+	if (!data_field) {
 		return -1;
 	}
 
 	// allocating the aspa_record itself
 	struct ordered_dyn_array *vector = (struct ordered_dyn_array *)lrtr_malloc(sizeof(struct ordered_dyn_array));
 
 	// malloc for aspa_record failed hence we return an error
-	if (vector == NULL) {
+	if (!vector) {
```

Comment:
I actually prefer `x == NULL` because its very verbose and easy to read understand

Reply:
ok might do that for all instances where I checked ptrs for `NULL`